spec/controllers/admin_users_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require "rails_helper"
+
+describe Admin::UsersController do
+  context "GET #new" do
+    context "when invalid params" do
+      it "won't create a new user" do
+        user = create(:user, admin: true)
+        allow(controller).to receive(:current_user).and_return(user)
+
+        admin_user_params = { email: "john@example.com" }
```

Comment:
This is again slightly confusing, since we are not really creating an admin here. So I would rename that to `user_params`.... and remove line 9 (no break)

Reply:
Done in sha: 281932b